src/main/flight/gps_rescue.c

```diff
@@ -484,6 +484,8 @@ void updateGPSRescueState(void)
 
     rescueState.isAvailable = checkGPSRescueIsAvailable();
 
+    static uint16_t descentDistance;
```

Comment:
Can you please move this to the beginning of the function?

Reply:
Done!